projects/common-library-typescript/package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "bs58": "^4.0.1",
     "grpc": "^1.8.4",
-    "grpc-caller": "^0.4.0",
+    "grpc-caller": "https://github.com/electricmonk/grpc-caller",
```

Comment:
Much safer to enforce a specific version (aka git version tag)

Reply:
why? it's my fork

Reply:
Sure. but even your fork might be updated at some point.

Reply:
whereupon I'll want the project to get the most recent version, no?